ota-plus-web/docs/modules/ROOT/pages/get-access.adoc

```diff
@@ -29,4 +29,4 @@ When you log in for the first time, to continue using the application, do the fo
 2. In the trial notification,  agree to the HERE Location Platform Services terms and conditions and privacy policy, and then click **Continue**.
 +
 [.lightbackground]
-image::img::log_in_trial.png[image]
+image::img::log_in_trial.png[image,300]
```

Comment:
Here too, better to use percentages than pixels.

Reply:
Changed